install/kubernetes/helm/istio/charts/sidecar-injector/templates/webhook.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1beta1" -}}
```

Comment:
Having this capability in k8s doesn't mean user wants to enable this web hook because it is an alpha feature.  FYI

Reply:
It need to also have sidecarInjector.enabled. is it possible to have sidecar injector without webhook? If it is then another flag is indeed needed

Reply:
I don't think you could have side car injection without web hook, however, if user is on k8s 1.9, user may choose not want to have side car injector enabled, which in the case, there is no need to apply the web hook.yaml.